_data/footer.yaml

```diff
@@ -0,0 +1,61 @@
+socialLinks:
+  - icon: linkedin
+    href: https://www.linkedin.com/company/red-hat
+    content: LinkedIn
+  - icon: youtube
+    href: https://www.youtube.com/user/RedHatVideos
+    content: Youtube
+  - icon: facebook
+    href: https://www.facebook.com/redhatinc
+    content: Facebook
+  - icon: twitter
+    href: https://twitter.com/RedHat
+    content: Twitter
+  - icon: github
+    href: https://github.com/redhat-israel
+    content: GitHub
+
+links:
+  - heading: המוצרים שלנו
+    links:
+    - content: Red Hat Ansible Automation Platform
+      href: https://www.redhat.com/en/technologies/management/ansible
+    - href: https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux
+      content: Red Hat Enterprise Linux
+    - href: https://www.redhat.com/en/technologies/cloud-computing/openshift
+      content: Red Hat OpenShift
+    - href: https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation
+      content: Red Hat OpenShift Data Foundation
+    - href: https://www.redhat.com/en/technologies/linux-platforms/openstack-platform
+      content: Red Hat OpenStack Platform
+    - href: https://www.redhat.com/en/technologies/all-products
+      content: See all products
+  
+  - heading: צרו קשר
```

Comment:
זה לא נראה כמו `המוצרים שלנו` (מבחינת style). לא יודע למה

Reply:
נראה לי אותו דבר, וה computed styles זהים

Reply:
In the attached photo [rh-website-view.png](https://postimg.cc/H8gvQJSY) we can see that the font of 
 "כתבו לנו", "שלחו לנו פידבק", ו"עקבו אחרינו ברשתות החברתיות"
looks different than the font of "Red Hat Ansible Automation Platform", "Red Hat Enterprise Linux", "Red Hat OpenShift", "Red Hat OpenShift Data Foundatio", "Red Hat OpenStack Platform", and  "See all products".

By looking at the markdown code it is looks similar, I guess that it uses different style/font for Hebrew and English, and therefore it looks different.